src/components/CircularProgress/CircularProgress.module.css

```diff
@@ -0,0 +1,22 @@
+.svg {
+  width: 100%;
+  height: 100%;
+  transform: rotate(-90deg);
+  overflow: visible;
+}
+
+.circleTransition {
+  transition: stroke-dashoffset 1s ease-in-out;
+}
+
+.label {
+  font-size: var(--font-size-300);
+  position: absolute;
```

Comment:
The parent wrapper would need `position: relative` for this to work reliably, wont it?

Reply:
Yes! Well seen 😄